chapter_1/2_kcenters.tex

```diff
@@ -1,3 +1,18 @@
+\documentclass{article}
```

Comment:
Remove these lines (1 to 15) and compile via the README's instructions. Adding these lines makes compilation fail.